.eslintrc.json

```diff
@@ -100,8 +118,11 @@
             "allowTemplateLiterals": true
           }
         ],
+        "@typescript-eslint/promise-function-async": "error",
```

Comment:
I don't love this rule myself, as it results in awkward stuff like:

```typescript
await Promise.all(
       [...this.models.values()].map((model) => this.updateSchema(model))
       [...this.models.values()].map(async (model) => this.updateSchema(model))
     );
```

Reply:
I can remove that rule.